eng/versioning/external_dependencies.txt

```diff
@@ -1,3 +1,8 @@
+# To avoid dependency conflicts between SDK and Spring, if you want to modify or add dependencies
+# that are managed in sdk/spring/scripts/spring_managed_external_dependencies.txt, please discuss
+# with Spring Integration team. Thanks
+# alias: rujche, xiada, yiliu6, zhihaoguo
```

Comment:
```
# To avoid dependency conflicts between SDK and Spring, when modify or add dependencies,
# make sure the version is same to sdk/spring/scripts/spring_managed_external_dependencies.txt.
# If your version is different from sdk/spring/scripts/spring_managed_external_dependencies.txt, 
# please discuss with Spring Integration team members:
# @chenrujun (rujche), @saragluna (xiada), @yiliuTo (yiliu6), @backwind1233 (zhihaoguo).  
# Thanks.
```